actions/profile/edit.php

```diff
@@ -51,8 +51,8 @@ function profile_array_decoder(&$v) {
 		forward(REFERER);
 	}
 
-	if ($value && $valuetype == 'url' && !preg_match('~^https?\://~i', $value)) {
-		$value = "http://$value";
+	if ($value && $valuetype == 'url') {
+		$value = elgg_normalize_url($value);
```

Comment:
hmmm there's a point, maybe we can make an `elgg_normalize_url($value, $strip_unsafe_true)`